roles/postfix-auth/templates/sender-canonical

```diff
@@ -1 +1 @@
-/^.*gst.hamburg.adfc.de$/   {{ postfix_host_email }}
+/.+/   {{ postfix_host_email }}
```

Comment:
Ich mag die alte Regex lieber, die erstetzt alle E-Mails die eine Absender-Doamin gst.hamburg.adfc.de hat, durch den Client.

Die neue ersetzt alles. Heißt aber auch, man kann keinen Absender mehr vorgeben, der nicht umgeschrieben wird.
Wenn also jemand z.B. aus dem Bankprogramm was als Hans.Wurst@hamburg.adfc.de verschicken will, geht das nicht mehr.

Reply:
Zwei Punkte hierzu:
1. Der hostname der kubuntu Maschine ist lediglich kubuntu-vm, hier triggert also die vorherige regex nicht bei der verwendiung von {{ ansible_fqdn }}, da jedoch "kubuntu-vm" auch keine vollständige Email Adresse ist, lehnen viele Mailserver die Mail nun ab.

2. Korrigier mich wenn ich hier falsch liege. Sollte ein Mailserver Emails nicht ablehnen wenn ein Authentifizierter Benutzer versucht Emails mit einer anderen als der eigenen Absenderadresse zu versenden? Ich weiß, unserer tut das nicht, aber das heißt ja nichts.

Reply:
> Zwei Punkte hierzu:
> 
> 1. Der hostname der kubuntu Maschine ist lediglich kubuntu-vm, hier triggert also die vorherige regex nicht bei der verwendiung von {{ ansible_fqdn }}, da jedoch "kubuntu-vm" auch keine vollständige Email Adresse ist, lehnen viele Mailserver die Mail nun ab.

Dann sollte man den Hostname in der main.cf ändern. Ich hoffe ja noch darauf, dass wir irgendwann IPV6 und statische IPs haben und Rechnernamen wirklich eingetragen sind im DNS....

> 2. Korrigier mich wenn ich hier falsch liege. Sollte ein Mailserver Emails nicht ablehnen wenn ein Authentifizierter Benutzer versucht Emails mit einer anderen als der eigenen Absenderadresse zu versenden? Ich weiß, unserer tut das nicht, aber das heißt ja nichts.

Naja, dass kann der Postmaster selbst bestimmen. Dagegen sprechen so Dinge wie Mailinglisten. jemand@example.org schickt etwas an akcomputer@... und der Mailserver macht daraus jemand@example.org -> martin@andere-domain... Schwups hast du eine E-Mail die nicht vom Mailserver kommt, aber über ihn verschickt wird.